backend/src/sync.js

```diff
@@ -85,6 +86,18 @@ async function saveBlocks(blocksInfo) {
                   };
                 })
               );
+              models.Account.bulkCreate(
+                blockInfo.transactions.map(tx => {
+                  return {
+                    accountId: tx.receiver_id,
+                    transactionHash: tx.hash,
+                    timestamp
+                  };
+                })
+              ).catch(function(err) {
```

Comment:
If account creation fails here, we will never get back to it in the future as we just let it continue. We should implement a proper fallback or at least do not mask the issue with `catch`, and let it crash.

Reply:
changed